drivers/i2c/i2c_shell.c

```diff
@@ -270,6 +270,36 @@ static int cmd_i2c_read(const struct shell *shell_ctx, size_t argc, char **argv)
 	return ret;
 }
 
+/* i2c speed <device> <speed>
+ * For: speed see constants like I2C_SPEED_STANDARD
```

Comment:
From the user perspective, it would be nice to be able to use one of the `I2C_SPEED_*` symbols here instead a value (which needs to be looked up in documentation).